src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from "react";
+import { fetchArticle } from "../../api";
+import { useParams } from "react-router-dom";
+import styles from "../css/SingleArticle.module.css";
+
+function SingleArticle() {
+  const [singleArticle, setSingleArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true)
+
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    fetchArticle(article_id).then(({ articles }) => {
+      setSingleArticle(articles[0]);
```

Comment:
Ideally, a single should be coming back as an object rather an an array from your Back End. No need to change now but please take a look at this after the course.